jumpscale/clients/stellar/stellar.py

```diff
@@ -759,3 +765,87 @@ def check_transaction_has_amount(self, transaction_hash):
         response = endpoint.call()
         results = response["_embedded"]["records"][0]
         return "amount" in results.keys()
+
+    def get_asset_by_code(self, code="TFT") -> stellar_sdk.Asset:
```

Comment:
I would make it get_asset with Issuer=None, known assets like TFT and XLM do not need an issuer, this way users do not need to interact with the stellar sdk themselves if it's a different asset